### PR TITLE
Add Managed Apps Upgrades dashboard

### PR DESCRIPTION
**Pull request by rossf7**
Towards https://github.com/giantswarm/roadmap/issues/322

This PR

- Adds Managed Apps Upgrades dashboard.

### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

File: helm/dashboards/dashboards/shared/managed-apps-upgrades.json [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+          "postEnd": false,
+          "postHeartbeat": false,
+          "postStart": true,
+          "server": "/analytics-plugin/write",
+          "showDetails": false
+        }
+      },
+      "title": "  ",
+      "transparent": true,
+      "type": "macropower-analytics-panel"
```
Comment (rossf7) [inline helm/dashboards/dashboards/shared/managed-apps-upgrades.json:46]: I tried to add the stats panel  but I get this error `Failed to load resource: net::ERR_BLOCKED_BY_CLIENT`

Is there any other config that is needed?

File: helm/dashboards/dashboards/shared/managed-apps-upgrades.json [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+      "type": "table"
+    }
+  ],
+  "refresh": "1m",
+  "schemaVersion": 30,
+  "style": "dark",
+  "tags": [],
+  "tags": [
+    "owner:team-batman",
+    "topic:managed-apps"
```
Comment (rossf7) [inline helm/dashboards/dashboards/shared/managed-apps-upgrades.json:240]: Are these topics OK?

File: helm/dashboards/dashboards/shared/managed-apps-upgrades.json [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+      },
+      "id": 6,
+      "options": {
+        "showHeader": true
+      },
+      "pluginVersion": "8.0.3",
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "sum by (app, catalog, latest_version, namespace, version) (aggregation:giantswarm:app_upgrade_available)",
```
Comment (rossf7) [inline helm/dashboards/dashboards/shared/managed-apps-upgrades.json:142]: The sum by is needed for when an app is installed multiple times on a cluster.

**Review by rossf7 — COMMENTED** [review]
Screen Shot 2021-09-06 at 12 42 41 PM



Tested on ghost

**Review by QuentinBisson — APPROVED** [review]

**Review by mig4 — APPROVED** [review]
Nice! Thanks for adding the analytics panel as well 👍 🏆

**Pull request merged**